caffe2/core/macros.h.in

```diff
@@ -76,6 +76,7 @@ static_assert(
   {"USE_EIGEN_FOR_BLAS", "${CAFFE2_USE_EIGEN_FOR_BLAS}"}, \
   {"USE_LITE_PROTO", "${CAFFE2_USE_LITE_PROTO}"}, \
   {"USE_MKL", "${CAFFE2_USE_MKL}"}, \
+  {"USE_MKLDNN", "${CAFFE2_USE_IDEEP}"}, \
```

Comment:
Do you want to change `CAFFE2_USE_IDEEP` to `CAFFE2_USE_MKLDNN`? I'd like these to be consistent.